_posts/2023-11-23-stork-latest-news.adoc

```diff
@@ -0,0 +1,135 @@
+                ServiceDefinition.of(new StaticConfiguration().withAddressList(example),
+                        new RandomConfiguration(), new StaticRegistrarConfiguration()));
+    }
+}
+----
+
+It’s important to note that the choice between programmatic and declarative configuration often depends on the specific requirements and constraints of
+your application.
+
+
+== Service discovery and selection strategies provided as CDI beans.
+
+Users may prefer using a framework that leverages CDI mechanism to easily manage and inject dependencies and have a more testable and maintainable code.
+Stork can now do that. Starting from the 2.0.1 release, users can use the service discovery and load balancer as beans.
+For that, it looks for CDI beans during the initialization in addition to the SPI providers.
+It is worth mentioning that this enhancement also contributes to improving the Quarkus experience.
+
+
+== New Service Discovery approaches added.
+
+We are happy to announce a few added service discovery strategies leveraging the power of DNS and Knative from 1.2 and 1.4 releases.
+With Knative, Smallrye Stork introduces seamless service discovery through its serverless infrastructure.
+The Stork Knative service discovery implementation is very similar to the Kubernetes one.
+Stork will ask for https://knative.dev/docs/serving/reference/serving-api/#serving.knative.dev/v1.Service[Knative services] to the cluster instead of vanilla https://kubernetes.io/docs/concepts/services-networking/service/#service-resource[Kubernetes services] used by the Kubernetes implementation.
+Again, to do so, Stork uses https://github.com/fabric8io/kubernetes-client/blob/master/extensions/knative/client/src/main/java/io/fabric8/knative/client/KnativeClient.java[Fabric 8 Knative Client] which is just an extension of Fabric8 Kubernetes Client
+The DNS-based service discovery is also here to stay. When a service has registered one or more instances in a Domain Name System (DNS) server,
+Stork will be able to discover them by querying the DNS. This strategy is simple and widely used, so Stork could not fail to implement it.
+
+
+== New Sticky service selection strategy
+
+The Stork load balancer family has been improved with a new one: the sticky service selection implementation.
+
+The sticky service selection implemented by Stork refers to a strategy where a client "sticks" to a particular instance of a service until it fails,
+then it selects another one. It is also possible to configure a backoff period for specifying how long a failing service instance should be retried.
+This can be useful in scenarios where maintaining a consistent connection to the same instance is beneficial, such as when dealing with sessions or
+stateful applications.
+
+
+== Enhance the service instances cache expiration policy.
+
+From almost the very beginning, Stork has supported in-memory caching of discovered instances through the extension of the CachingServiceDiscovery class.
```

Comment:
```suggestion
Since almost the first release, Stork has provided in-memory caching of discovered instances by extending the `CachingServiceDiscovery` class.
```